### DISCUSSION TITLE
How do I make more items in a UniStore file?

### DISCUSSION THREAD

**Discussion by ghost** [Universal-Team/Universal-Updater#106] [Q&A]
My file is here: https://github.com/GeTuck/UniHB/raw/main/Builds/UniHB-v015.unistore but when I open it up in the UniStore, there is only one item, that is Minicraft (I scan the QR code in my release). I tried to add another thing, but I couldn't find anything in the UU wiki, so I tried to imrovise and it didn't work. How can I have more than 1 item?

EDIT: Also, how can I make a spritesheet? I can't download the spritesheet maker because I don't have Windows, can I do it online somehow?

**Comment by Epicpkmn11** [comment]
Your lines 36-39 are wrong, here's a fixed one: https://gist.github.com/Epicpkmn11/16826d3b19731119c1dac1bd018aa5c4 ([diff](https://gist.github.com/Epicpkmn11/16826d3b19731119c1dac1bd018aa5c4/revisions#diff-450060e120b7cf7752663e1e20d2d76205c15a868df08037090c11f54598a8e0L36-R37)) (I also fixed the indentation)

For the spritesheet you'll need a computer that can run [tex3ds](https://github.com/devkitPro/tex3ds), afaik there's not an easy way to run it online. One way you could do is what [Universal-DB](https://github.com/Universal-Team/db) does and have GitHub Actions run tex3ds, you could even just fork Universal-DB and edit `source/source.json` to your apps, then as long as Actions is enabled in your repository it should simply generate the UniStore and spritesheet for you with every commit.

Edit: Also fyi converted your issue to a discussion as it's just a question not a problem or feature request or such for Universal-Updater so this works better.

**Reply by ghost** [reply]
thx
